ConsoleApp/Program.cs

```diff
@@ -0,0 +1,19 @@
+﻿int[] BubbleSort (int[] arr)
+{
+    var copy = new int[arr.Length];
+    Array.Copy(arr, copy, arr.Length);
+    var iterations = copy.Length;
```

Comment:
minor, copy length is just getter, it's not critical to use this value in condition
critical to use function result cause each time you need to calculate the value and it can be a complex algorithm
here it's ok :)